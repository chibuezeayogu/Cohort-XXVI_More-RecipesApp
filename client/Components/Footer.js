import React, { Component }from 'react';
import PropTypes from 'prop-types';
import { Footer } from 'react-materialize';


class BottomFooter extends  Component {
    render() {
        return (
            <div>
                <Footer copyrights="&copy 2017 Copyright Text"
                moreLinks={
                    <a className="grey-text text-lighten-4 right" href="#!">Andela Project by Ayogu Chibueze></a>
                }
                
                className='black'
            >
                    
                </Footer>;
            </div>
        );
    }
}

BottomFooter.propTypes = {
    logout: PropTypes.func.isRequired,
    history: PropTypes.func.isRequired,
    push: PropTypes.func.isRequired,
    className: PropTypes.string,
    copyrights: PropTypes.string,
};

export default BottomFooter;