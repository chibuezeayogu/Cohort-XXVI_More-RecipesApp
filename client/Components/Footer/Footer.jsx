import React from 'react';

/**
 * constant page footer
 * @constant
 * @returns {void}
 */
const Footer = () => (
  <footer className="page-footer black footer">
    <div className="container">
      <div className="row">
        <div className="col l6 s12">
          <h6 className="white-text"> Â© 2018 Copyright</h6>
        </div>
        <div className="col l4 offset-l2 s12">
          <h6 className="white-text">Andela Project by Ayogu Chibueze</h6>
        </div>
      </div>
    </div>
  </footer>
);

export default Footer;
