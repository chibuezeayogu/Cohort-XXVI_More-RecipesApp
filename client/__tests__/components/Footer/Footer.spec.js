import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import Footer from '../../../Components/Footer/Footer';

describe('Footer', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
