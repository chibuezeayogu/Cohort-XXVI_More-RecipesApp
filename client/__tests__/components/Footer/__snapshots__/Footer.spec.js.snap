// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  "node": <footer
    className="page-footer black footer"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col l6 s12"
        >
          <h6
            className="white-text"
          >
             © 2018 Copyright
          </h6>
        </div>
        <div
          className="col l4 offset-l2 s12"
        >
          <h6
            className="white-text"
          >
            Andela Project by Ayogu Chibueze
          </h6>
        </div>
      </div>
    </div>
  </footer>,
  "nodes": Array [
    <footer
      className="page-footer black footer"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col l6 s12"
          >
            <h6
              className="white-text"
            >
               © 2018 Copyright
            </h6>
          </div>
          <div
            className="col l4 offset-l2 s12"
          >
            <h6
              className="white-text"
            >
              Andela Project by Ayogu Chibueze
            </h6>
          </div>
        </div>
      </div>
    </footer>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Footer />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <footer
          className="page-footer black footer"
        >
          <div
            className="container"
          >
            <div
              className="row"
            >
              <div
                className="col l6 s12"
              >
                <h6
                  className="white-text"
                >
                   © 2018 Copyright
                </h6>
              </div>
              <div
                className="col l4 offset-l2 s12"
              >
                <h6
                  className="white-text"
                >
                  Andela Project by Ayogu Chibueze
                </h6>
              </div>
            </div>
          </div>
        </footer>,
        "_debugID": 2,
        "_renderedOutput": <footer
          className="page-footer black footer"
        >
          <div
            className="container"
          >
            <div
              className="row"
            >
              <div
                className="col l6 s12"
              >
                <h6
                  className="white-text"
                >
                   © 2018 Copyright
                </h6>
              </div>
              <div
                className="col l4 offset-l2 s12"
              >
                <h6
                  className="white-text"
                >
                  Andela Project by Ayogu Chibueze
                </h6>
              </div>
            </div>
          </div>
        </footer>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Footer />,
}
`;
